sky/cli.py

```diff
@@ -2752,10 +2771,29 @@ def _down_or_stop_clusters(
                     f'{names_str} is currently not supported.\n'
                     'Please omit the cluster(s) with reserved prefix '
                     f'{name_to_reserved_prefix.values()}.')
-            raise click.UsageError(
-                f'{operation} cluster(s) with reserved prefix '
-                f'{reserve_prefix_str} is not supported. To teardown a '
-                'service, please use `sky serve down`.')
+            if not down:
+                raise click.UsageError(
+                    f'{operation} cluster(s) with reserved prefix '
+                    f'{reserve_prefix_str} is not supported. To teardown a '
+                    'service, please use `sky serve down`.')
```

Comment:
just to confirm. where did you distinguish spot controller and serve controller?

Reply:
The `name_to_reserved_prefix` contains all cluster filtered with `backend_utils.SKY_RESERVED_CLUSTER_PREFIXES`, and the `reserved_clusters` contains all cluster filtered with `backend_utils.SKY_RESERVED_CLUSTER_NAMES`.